pkg/driver/driver_openstack.go

```diff
@@ -468,3 +471,31 @@ func (d *OpenStackDriver) GetVolNames(specs []corev1.PersistentVolumeSpec) ([]st
 	}
 	return names, nil
 }
+
+func waitForStatus(c *gophercloud.ServiceClient, id, pending []string, target []string, secs int) error {
```

Comment:
Can't we use `time.Duration` instead of int for seconds?

Reply:
```suggestion
func waitForStatus(c *gophercloud.ServiceClient, id string, pending []string, target []string, secs int) error {
```
Otherwise make build fails.

Reply:
fixed the typo. I expected you have some basic travis tests.